docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -2235,3 +2235,24 @@ If the amount is not in hundreds, it could be a problem. A possible workaround i
 export JAVA_OPTS=-Djava.security.egd=/dev/urandom
 ----
 The proper solution is to increase the entropy available for the system. That is specific for each OS vendor and virtualization solution though.
+
+=== Work around missing CPU features
+
+When building on recent machines and running your native executable on older machines, you may see the following failure when starting the application:
+
+----
+The current machine does not support all of the following CPU features that are required by the image: [CX8, CMOV, FXSR, MMX, SSE, SSE2, SSE3, SSSE3, SSE4_1, SSE4_2, POPCNT, LZCNT, AVX, AVX2, BMI1, BMI2, FMA].
+Please rebuild the executable with an appropriate setting of the -march option.
+----
+
+The native compilation used more advanced instruction sets, not supported by the CPU running the application.
+To work around that issue, add the following line to the `application.properties`:
+
+[source, properties]
+----
+quarkus.native.additional-build-args=-march=compatibility
```

Comment:
`-march=compatibility` is perhaps too much. How about `-march=x86-64-v2` instead?

Reply:
Note, that `x86-64-v2` is AMD64 specific. Since Quarkus also supports AArch64 `compatibility` seems more appropriate (as a generic workaround).

Reply:
I will add a note about that.
I wanted to avoid listing the possibilities as they may change in the future and can be confusing.